standalone/src/main/scala/io/delta/standalone/internal/DeltaLogImpl.scala

```
@@ -210,19 +242,35 @@ private[internal] class DeltaLogImpl private(
    * allowed to read the table that is using the given `protocol`.
    */
   def assertProtocolRead(protocol: Protocol): Unit = {
+    // Check Delta Standalone's compatibility
     if (protocol != null && Action.readerVersion < protocol.minReaderVersion) {
       throw new DeltaErrors.InvalidProtocolVersionException(Action.protocolVersion, protocol)
     }
+    // Check the connector's compatibility
+    if (!ignoreConnectorProtocolChecks) {
+      if (protocol != null && connectorReaderVersion < protocol.minReaderVersion) {
+        throw new DeltaErrors.InvalidProtocolVersionException(
```

we should throw a different error. this error references `Delta Standalone`

---

nvm saw your comment below. can you include a `TODO` here too (about using a different error msg)? So Venki/the next reviewer doesn't repeat what I did